packet_service/types.lua

```diff
@@ -1887,7 +1947,7 @@ types.incoming[0x0DD] = struct({
     mp                  = {0x08, uint32},
     tp                  = {0x0C, uint32},
     flags               = {0x10, uint16}, -- #BYRTH# Should be a bitfield
-    index               = {0x14, entity_index},
+    player_index        = {0x14, entity_index},
```

Comment:
I would actually suggest removing the `player_` both here and for `player_id`, same for `0x0DC`. Imo it's obvious since the packet is entirely about party members, so everything in there refers to them, also I like having `player` refer mostly to the current player, not any PC.

Reply:
The `player_` prefix has always annoyed me because `player_` character (PC) is any player.

makes more sense to change the ones exclusive to yourself to be `self_` than change this to be our only uninformative `index` name.

That said, it seems like writing this comment was probably about as hard as changing it would have been. If there's no room for discussion, just change it next time.